mk/spksrc.wheel.mk

```diff
@@ -82,6 +77,11 @@ build_wheel_target: $(PRE_WHEEL_TARGET)
 				. $(CROSSENV) && $(RUN) _PYTHON_HOST_PLATFORM="$(TC_TARGET)" CFLAGS="$(CFLAGS) -I$(STAGING_INSTALL_PREFIX)/$(PYTHON_INC_DIR) $(WHEELS_CFLAGS)" LDFLAGS="$(LDFLAGS) $(WHEELS_LDFLAGS)" $(PIP_WHEEL) --no-build-isolation --requirement $(WHEELHOUSE)/$(WHEELS_CROSS_COMPILE) ; \
 			fi ; \
 		fi ; \
+		if [ -f "$(WHEELHOUSE)/$(WHEELS_PURE_PYTHON)" ]; then \
+			$(MSG) "Force pure-python" ; \
+			export LD= LDSHARED= CPP= NM= CC= AS= RANLIB= CXX= AR= STRIP= OBJDUMP= READELF= CFLAGS= CPPFLAGS= CXXFLAGS= LDFLAGS= && \
+				$(RUN) $(PIP_WHEEL) --requirement $(WHEELHOUSE)/$(WHEELS_PURE_PYTHON) ; \
+		fi ; \
```

Comment:
Just a thought, I had notice wheel build failures when pure was after cross but that was quite a few commits ago and maybe unrelated (and in theory it really shouldn't affect it at all).  Adding this comment just to keep track in case it bytes us later.

Reply:
I will have to check the logs of the failed builds (evansport-7.0, hi3535-6.1) whether it is related to this.